mcpserver/mcpserver.go

```diff
@@ -0,0 +1,107 @@
+package mcpserver
+
+import (
```

Comment:
🐛 **Bug (High):** Missing import for 'fmt' package. The file uses fmt.Errorf and fmt.Sprintf functions but doesn't import the fmt package.

```suggestion
import (
	"context"
	"fmt"

	"dagger.io/dagger"
	"github.com/dagger/container-use/environment"
	"github.com/dagger/container-use/repository"
	"github.com/mark3labs/mcp-go/mcp"
)
```